docs/notebooks/Berger_POPC_25ns_buildH_calcOP.ipynb

```diff
@@ -0,0 +1,1394 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "gamma1_1 POPC C1  H11\n",
+      "gamma1_2 POPC C1  H12\n",
+      "gamma1_3 POPC C1  H13\n",
+      "gamma2_1 POPC C2  H21\n",
+      "gamma2_2 POPC C2  H22\n",
+      "gamma2_3 POPC C2  H23\n",
+      "gamma3_1 POPC C3  H31\n",
+      "gamma3_2 POPC C3  H32\n",
+      "gamma3_3 POPC C3  H33\n",
+      "beta1 POPC C5  H51\n"
+     ]
+    }
+   ],
+   "source": [
+    "!head order_parameter_definitions_MODEL_Berger_POPC.def"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The first column is the generic C-H name, second is the residue (lipid) name, third column the carbon PDB name, last column the hydrogen PDB name. This file will tell buildH which Hs you want to rebuild, and then to calculate the order parameter on the corresponding C-Hs. It means that it is possible to provide only a subset of possible C-Hs if, for example, you only want those from the polar heads. Note that if you want to output a trajectory (not done in this notebook), the def file must contain **all** possible C-Hs.\n",
+    "\n",
+    "Now, let's have a quick look to the `.gro` file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Generated by trjconv : 2:1:1 sdpc:SM:CHOL system 256 lipids t= 50000.00000\n",
+      "28526\n",
+      "    1PLA     C1    1   5.576   3.681   2.227 -0.0394  0.0935 -0.3339\n",
+      "    1PLA     C2    2   5.553   3.886   2.337  0.1128 -0.0058 -0.1560\n",
+      "    1PLA     C3    3   5.762   3.834   2.277 -0.6144  0.1201 -0.8703\n",
+      "    1PLA     N4    4   5.640   3.768   2.326 -0.1182 -0.1907 -0.0302\n",
+      "    1PLA     C5    5   5.658   3.709   2.460 -0.2986 -0.2323 -0.0245\n",
+      "    1PLA     C6    6   5.766   3.607   2.495  0.4136  0.5471  0.0677\n",
+      "    1PLA     O7    7   5.752   3.488   2.417  0.3142  0.2538  0.5323\n",
+      "    1PLA     P8    8   5.893   3.411   2.413  0.2220  0.1104  0.0738\n"
+     ]
+    }
+   ],
+   "source": [
+    "!head endCONF.gro"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that in this `.gro` file containing POPC lipids, the residue name is called `PLA`. This residue name will be of importance when launching buildH.\n",
+    "\n",
+    "Great, all files were successfully downloaded, we can now proceed. Let's launch buildH!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Launching buildH for order parameter calculation\n",
+    "\n",
+    "Now, we can launch buildH on this downloaded trajectory. For that we need a few arguments :\n",
+    "\n",
+    "- the topfile corresponding to the system passed without argument: it consists of a pdb or gro file, here `endCONF.gro`, buildH infers the topology of the system from it;\n",
```

Comment:
Pas de flag pour le fichier de topologie ? pourquoi ce choix ?

Reply:
Now it's implemented, thanks for the suggestion!